dataland-frontend/tests/e2e/specs/search-company-framework-data/SearchCompaniesForFrameworkData.ts

```diff
@@ -171,7 +171,9 @@ describe("As a user, I expect the search functionality on the /companies page to
     getKeycloakToken(uploader_name, uploader_pw).then((token) => {
```

Comment:
Previsit.ts file is empty, can it be deleted?

Reply:
It is not there anymore. Why do you still see it?

Reply:
Good question but that then seems like a github bug. For me the question is answered and i resolve the comment.